src/vs/workbench/contrib/terminal/browser/media/terminal.css

```diff
@@ -245,7 +245,7 @@
 	top: 0;
 }
 
-.monaco-workbench .pane-body.integrated-terminal .xterm .xterm-helper-textarea:focus {
+.monaco-workbench .xterm .xterm-helper-textarea:focus {
```

Comment:
This wasn't happening for `.editor-instance` before, not sure if that's an issue?